styles/packages.less

```diff
@@ -55,7 +55,7 @@
   }
 
   .input-block-item {
-    flex: 1 1 @ui-size*10;
+    flex: 1 1 @ui-size*20;
```

Comment:
Yeah, it already does and works for most themes. This is just necessary because the One theme uses `em`s for font-size to make everything resizable.